src/components/UserList/UserListToolbar.tsx

```diff
@@ -35,8 +32,9 @@ export function UserListToolbar(props: UserListToolbarProps) {
       )}
       {numSelected > 0 ? (
         <>
-          {selectedRows.every((row) => {
-            return row.userType === 'Pending';
+          {selected.every((row) => {
+            /*back doesn't have a pending type yet, this will need to be changed later*/
+            return row.rolesId === '2';
```

Comment:
this should be an enum. You can probably get this type from generated open API types